test/e2e/chart/Chart.yaml

```diff
@@ -0,0 +1,34 @@
+#  Copyright (C) 2018 Yunify, Inc.
```

Comment:
are you adding the qingcloud-csi chart here? Why not using the existing chart?

Reply:
In the test environment, there are likely already a deployed qingcloud-csi, with some of the cluster-level components like `ClusterRole` and `ClusterRoleBinding` created in a static name, which results in a conflict when deploying the qingcloud-csi driver of the e2e test.

So I'm adding a local chart with the names modified to a configurable template, to avoid conflicts.

Reply:
okay, I understand. But adding a bunch of source files from another repo to this repo is strongly discouraged. They work now, but maintaining them in the future will be a nightmare.

Try to use the existing qingcloud-csi chart ( or even modify the existing qingcloud-csi chart to support the e2e test), but don't add the chart source codes to this repo.

The simplest and quickest way to me may be adding a prerequisite: this e2e test need a clean cluster without qingcloud-csi deployed. if it's deployed, uninstall it before proceed.

Reply:
Yeah, I agree on that, will just leave it to the user in the prerequisites doc section.